cli-test/spec/spec_helper.rb

```
@@ -20,16 +28,15 @@
   end
 
   config.before(:each) do
-    Dotenv.load('.env')
-    run_command_and_stop('oxen config --name ruby-test --email test@oxen.ai')
-    run_command_and_stop("oxen config --auth dev.hub.oxen.ai #{ENV['OXEN_API_KEY']}")
-    run_command_and_stop('oxen delete-remote --name EloyMartinez/performance-test --host dev.hub.oxen.ai -y',
-                         fail_on_error: false)
+    regexp = ".env".force_encoding('UTF-16LE')
```

RuboCop (1.69.1)

[warning] Useless assignment to variable - `regexp`.

(Lint/UselessAssignment)